mobx_codegen/lib/src/template/store.dart

```diff
@@ -61,10 +61,16 @@ abstract class StoreTemplate {
   $actions''';
 
     if (generateToString) {
+      final publicObservablesList = observables.templates
+      ..removeWhere((element) => element.isPrivate);
+
+      final publiccomputedsList = computeds.templates
```

Comment:
camel casing is off. Should be `publicComputedsList`, 'C' capitalized

Reply:
:-(